### PR TITLE
Handle click track item

### PR DESCRIPTION
**Pull request by ngochieu1297**
## [Pull request title]
Handle click track item
### [Redmine link]
https://edu-redmine.framgia.vn/issues/7530

### CODE REVIEW

**Comment by ngochieu1297** [comment]
ready

File: app/src/main/java/com/example/music_58/ui/main_play/MainPlayActivity.java [hunk 1]
```diff
@@ -1,22 +1,92 @@
...
+        mImagePlay = findViewById(R.id.image_play);
+        mImageDownload = findViewById(R.id.image_download);
+        mImageFavorite = findViewById(R.id.image_favorite);
+        mImageLoop = findViewById(R.id.image_loop);
+        mImageShuffle = findViewById(R.id.image_shuffle);
+    }
+
+    private void initAnimation() {
+        ObjectAnimator animator = ObjectAnimator.ofFloat(mImageArtwork,
+                "rotation",0, 360);
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/example/music_58/ui/main_play/MainPlayActivity.java]: hardcode
format code

File: app/src/main/java/com/example/music_58/ui/main_play/MainPlayActivity.java [hunk 2]
```diff
@@ -1,22 +1,92 @@
...
+        animator.setDuration(30000);
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/example/music_58/ui/main_play/MainPlayActivity.java]: tương tự

File: app/src/main/java/com/example/music_58/ui/main_play/MainPlayActivity.java [hunk 3]
```diff
@@ -1,22 +1,92 @@
...
+                break;
+        }
+    }
+
+    private void handleEvent() {
+        mImageBack.setOnClickListener(this);
+        mImagePlay.setOnClickListener(this);
+    }
+
+    private void initUI() {
```
Comment (NguyenMinhTuanB) [inline app/src/main/java/com/example/music_58/ui/main_play/MainPlayActivity.java]:
```suggestion
    private void initView() {
```

File: app/src/main/res/layout/activity_main_play.xml [hunk 4]
```diff
@@ -202,6 +203,7 @@
         android:id="@+id/image_loop"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
+        android:layout_marginBottom="4dp"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/activity_main_play.xml]: đây nữa

**Comment by ngochieu1297** [comment]
ready

**Pull request merged**